contracts/token/ERC20/extensions/draft-ERC20Bridgeable.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {ERC20} from "../ERC20.sol";
+import {ERC165, IERC165} from "../../../utils/introspection/ERC165.sol";
+import {IERC7802} from "../../../interfaces/draft-IERC7802.sol";
+
+/**
+ * @dev ERC20 extension that implements the standard token interface according to
+ * https://eips.ethereum.org/EIPS/eip-7802[ERC-7802].
+ */
+abstract contract ERC20Bridgeable is ERC20, ERC165, IERC7802 {
+    /// @dev Modifier to restrict access to the token bridge.
+    modifier onlyTokenBridge() {
+        // Token bridge should never be impersonated using a relayer/forwarder. Using msg.sender is preferable to
+        // _msgSender() here, both for cost and security reasons.
```

Comment:
```suggestion
```

Don't we say the costs are 0? Shouldn't contradict ourselves. Also given that we plan on removing context, i dont think we need the comment at all.

Reply:
The documentation infers that __msgSender() costs more than msg.sender even if it’s not overridden.

Reply:
iirc, the concern here was that the bridge shouldn't be changed or impersonated in the first place, so adding `_msgSender` here implied that it could be overridden, which is completely unexpected in the context of a token that's trusting a bridge for minting and burning

Reply:
No reason to do this post 7702